Update conf.py for current Sphinx

The CI failure in #32 is because the conf.py file is out of date with current Sphinx.  (need to comment the `language=` line now, the default is acceptable).

_Originally posted by @prjemian in https://github.com/bluesky/area-detector-handlers/issues/32#issuecomment-1263863704_

The error:

```
make: Entering directory '/home/runner/work/area-detector-handlers/area-detector-handlers/docs'
Running Sphinx v5.2.3

Warning, treated as error:
Invalid configuration value found: 'language = None'. Update your configuration to a valid language code. Falling back to 'en' (English).
make: *** [Makefile:20: html] Error 2
make: Leaving directory '/home/runner/work/area-detector-handlers/area-detector-handlers/docs'
Error: Process completed with exit code 2.
```

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -86,7 +86,7 @@ release = area_detector_handlers.__version__
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+# language = None
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.

